go.mod

```diff
@@ -52,7 +52,7 @@ require (
 	github.com/tomnomnom/linkheader v0.0.0-20180905144013-02ca5825eb80 // indirect
 	github.com/vektra/mockery v1.1.2
 	github.com/voxelbrain/goptions v0.0.0-20180630082107-58cddc247ea2 // indirect
-	github.com/weaveworks/goformation/v4 v4.10.2-0.20210202192510-c984c16fe84b
+	github.com/weaveworks/goformation/v4 v4.10.2-0.20210510110359-567e61bbcf4c
```

Comment:
This is using the `taints` branch: https://github.com/weaveworks/goformation/pull/27